src/circular-progress.jsx

```diff
@@ -72,15 +72,17 @@ const CircularProgress = React.createClass({
     this._rotateWrapper(wrapper);
   },
 
+  componentWillUnmount() {
+    clearTimeout(this._scalePathTimer);
```

Comment:
Hmmmm.... maybe it's more readable inside the component. `_` is conveniently private. So new people won't be confused. I for one realized [this](https://github.com/callemall/material-ui/blob/master/src/snackbar.jsx#L24) was private as soon as i saw it. I think it's not a bad idea to keep it this way, no?